omod/src/main/webapp/resources/scripts/app.js

```diff
@@ -0,0 +1,61 @@
+
+var app  = angular.module('Tag',['tagService','ngDialog','uicommons.common.error']);
+
+
+app.controller('tagCtrl',['$scope','TagService','ngDialog',function ($scope,TagService,ngDialog) {
+    $scope.init = function(personUuid) {
+            $scope.thisPersonUuid = personUuid;
+          $scope.loadTags =function () {
+              var payload = {
+                  objectType: 'org.openmrs.Person',
+                  objectUuid: personUuid
+              };
+              TagService.getTags(payload).then(function (res) {
+                  $scope.tags = res;
+              });
+          }
+         $scope.loadTags();
```

Comment:
But placing it inside init isn't the right way to do it, it's only working because you call it from the init function which is also wrong. Sometimes because it works doesn't mean it's the proper way to do something otherwise you can run into other obscure things down the road, probably this might explain why calling it from somewhere else fails, you need to move it out, it should be able to work if you call it properly i.e call it outside of the init function, it should be the last call in the main enclosing function you are passing to your angular module